src/Batcave/pbconfig.ts

```diff
@@ -0,0 +1,33 @@
+import { ContentRating, SourceInfo, SourceIntents } from "@paperback/types";
+
+export default {
+    name: "Batcave",
+    description: "Extension that pulls content from batcave.biz.",
+    version: "1.0.0-alpha.6",
+    icon: "icon.png",
+    language: "en",
+    contentRating: ContentRating.EVERYONE,
+    badges: [
+        {
+            label: "Aggregator",
+            textColor: "#FFFFFF",
+            backgroundColor: "#800080",
+        },
+        { label: "Mature", textColor: "#FFFFFF", backgroundColor: "#800080" },
+        {
+            label: "Western Comics",
+            textColor: "#FFFFFF",
+            backgroundColor: "#C71585",
+        },
+    ],
```

Comment:
No badges until the npm package has been released.
```suggestion
    badges: [],
```